source/blog/2018-07-19-ovn-dpdk.html.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Upgraded DPDK support in oVirt
+author: lgoldber
+tags: OVS-DPDK, Open vSwitch, OVN, ansible
+date: 2018-07-19 10:00:00 CET
+comments: true
+published: true
+---
+[DPDK](http://dpdk.org/) (Data Plane Development Kit) provides high-performance packet processing libraries and user space drivers.
+
+[Support for DPDK](https://www.ovirt.org/blog/2017/09/ovs-dpdk/) in oVirt was introduced in 2017, and is now upgraded in terms of both deployment via Ansible and usage via [Open Virtual Network](http://www.ovn.org/).
+
+While still experimental, OVN-DPDK in oVirt is now available in versions 4.2 and upwards.
+
+# What's changed?
+
+### Ansible DPDK host setup
+
+Host configuration for DPDK usage is now automated using Ansible. This primarly includes:
+* Hugepages configuration -- hugepage size and quantity in the kernel.
+* CPU partitioning.
+* Binding NICs to userspace drivers. 
+* OVS-DPDK related configuration (initialization, socket memory, pmd thread core binding, etc).
+
+The role is installed via Ansible galaxy:
```

Comment:
Worth mentioning that the playbook can run either on the engine or on the host.

Reply:
Redundant IMO, I also think that running the playbook on the host is impossible in case a reboot is required?

Reply:
Ok i'm convinced